site/content/documentation/tutorials/maven-eclipse-project.md

```diff
@@ -7,174 +7,180 @@ autonumbering: true
 If you are new to RDF4J, or to tools like Eclipse IDE or Apache Maven, this tutorial will help you get started.
 <!--more-->
 
-NOTE: using Maven or Eclipse is not required if you want to use RDF4J. These are simply very useful tools for quickly getting a Java project started. Maven is good because it allows you to just define which libraries you want to use and never worry about any further third-party libraries you might need, and Eclipse IDE is good because it has good integration with Maven, code completion features, and is just generally a great Java development environment.
+{{< info >}}
+You don't <i>have</i> to use Apache Maven or Eclipse IDE if you want to work with RDF4J. These are simply very useful tools for quickly getting a Java project started. Maven is good because it allows you to just define which libraries you want to use and never worry about any further third-party libraries you might need, and Eclipse IDE is good because it has good integration with Maven, code completion features, and is just generally a great Java development environment. But you can work with RDF4J just as well in a different build tool or IDE.
+{{</ info >}}
 
-In this tutorial, I assume that you have a basic understanding of programming in Java, and have at least an inkling of what RDF is. However, I do not assume that you know how to use either RDF4J, Maven, or Eclipse, so we’ll go through it all one step at a time. If anything is already sufficiently familiar to you, you are of course free to skip ahead!
+In this tutorial, we assume that you have a basic understanding of programming in Java, and have at least an inkling of what RDF is. However, we do not assume that you know how to use either RDF4J, Maven, or Eclipse, so we'll go through it all one step at a time. If anything is already sufficiently familiar to you, you are of course free to skip ahead!
 
 ## Setting up your environment
 
-Before we kick off, you will need to install the [Eclipse IDE](http://eclipse.org/). In this tutorial, we will use Eclipse for Java EE Developers version 4.5.2 (Mars.2), but it shouldn’t matter too much which exact version you have installed. Select either the “Eclipse for Java developers” or “Eclipse for Java EE developers” installation option.
+Before we kick off, you will need to install the [Eclipse IDE](http://eclipse.org/). In this tutorial, we will use Eclipse for Java Developer version 4.18.0 (2020-12), but it shouldn’t matter too much which exact version you have installed. Select either the "Eclipse for Java developers" or "Eclipse for Java EE developers" installation option.
```

Comment:
There is a minor typo here. Should be "Intellij IDEA" instead of "Eclipse". Don't worry, it's an easy mistake to make.

😈